_posts/2020-04-01-cockpit-216.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 216
+author: lis
+date: 2020-04-01
+tags: cockpit linux
+slug: cockpit-216
+category: release
+summary: Cockpit with improved SELinux page and last login banner
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 216.
+
+### SELinux: Automatic application of solutions
+
+The SELinux page now has "Apply this solution" buttons for suggested
+solutions to logged access control errors.
```

Comment:
Perhaps "The SELinux page can now automatically apply solutions that set [SELinux Booleans](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/selinux_users_and_administrators_guide/sect-security-enhanced_linux-working_with_selinux-booleans)." ?